tests/test_cli.py

```diff
@@ -100,29 +117,57 @@ def verify_docker_output(docker_cmd, expect_values):
     return True
 
 
-def wait_verify_docker_output(docker_cmd, expect_values):
+def wait_verify_docker_output(docker_cmd, expect_values, use_shell=False):
     times = 0
-    while (not verify_docker_output(docker_cmd, expect_values)):
+    while (not verify_docker_output(docker_cmd, expect_values, use_shell)):
         print("Waiting until docker command is ready... ...\n")
         time.sleep(10)
         times += 1
         if times > 360:
             raise Exception('Timeout to wait until it appears expected value ' + str(expect_values))
 
 
+def get_all_docker_volumes():
+    output = start_process(['docker', 'volume', 'ls'], False)
+    return output
+
+
 def remove_docker_volumes(volumes):
     for volume in volumes:
-        start_process(['docker', 'volume', 'rm', volume, '-f'], False)
+        if volume in get_all_docker_volumes():
```

Comment:
ge_all_docker_volumes() will be executed for each volume.  is it as expected?

Reply:
And why we need to check whether the volume exists? If not exists, does that mean there should be exception and cause the test case fail?

Reply:
Updated code